cli/initiator/reshare.go

```diff
@@ -0,0 +1,266 @@
+package initiator
```

Comment:
Its seems like this file shares a lot of code with `init.go`, can resharing be a subcommand of init? or at least lets break out some of the repeated code into functions, the purpose is that we won't have to update the same code twice (or forget to) if we do any changes.

Reply:
Honestly, I would rather keep it separate commands. I tried to have resharing as a subcommand, but it looks messy together. Ill separate functions as much as I can find